src/main.rs

```diff
@@ -22,85 +22,91 @@ mod exercise;
 mod run;
 mod verify;
 
+// In sync with crate version
+const VERSION: &'static str = "4.3.0";
```

Comment:
Constants by default have static lifetime so the lifetime does not need to be specified.

https://rust-lang.github.io/rust-clippy/master/index.html#redundant_static_lifetimes